data/json/vehicles/custom_vehicles.json

```diff
@@ -0,0 +1,114 @@
+[
+    {
+        "type" : "vehicle",
+        "id" : "hearse",
```

Comment:
At the time I looked to include it in what I thought might be an appropriate vehicle file but I didn't see one that it fit under... so I made a new one. The idea was it would be a file for vehicles like limousines, survivor vehicles, or other custom made vehicles. However, I hold not attachment to it as that you can't rename it as you wish. So you are most welcome to do so.